Backpack/src/main/res/values/backpack.button.xml

```diff
@@ -6,9 +6,9 @@
   </declare-styleable>
 
   <style name="bpkButtonBase">
-    <item name="android:fontFamily">sans-serif-light</item>
+    <item name="fontFamily">sans-serif-light</item>
```

Comment:
Are these changes intended?

Reply:
These are done by the migration tool.

Reply:
Removing the `android:` prefix means we are overriding `fontFamily` in the AppCompat Theme. If this is what we want then it's correct.